Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN make
 FROM amazonlinux:2 AS amazonlinux
```

Comment:
As discussed offline using https://hub.docker.com/r/amd64/amazonlinux/ reduced image size significantly (from 750MB to 250MB). We should move to that instead of using amazonlinux:2